ci-operator/step-registry/ipi/install/powervc/install/ipi-install-powervc-install-commands.sh

```
@@ -366,6 +366,20 @@ openshift-install --dir="${DIR}" create manifests
 
 sed -i '/^  channel:/d' "${DIR}/manifests/cvo-overrides.yaml"
 
+# Sets up the chrony machineconfig for the worker nodes
+CHRONY_WORKER_YAML="${SHARED_DIR}/99-chrony-worker.yaml"
+if [ -f "${CHRONY_WORKER_YAML}" ]; then
+  echo "Saving ${CHRONY_WORKER_YAML} to the install directory..."
+  cp ${CHRONY_WORKER_YAML} "${DIR}/manifests"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)